typos.toml

```diff
@@ -0,0 +1,20 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+[files]
+extend-exclude = [
+    "vendor/*",
+    "third_party/*",
+    "*generated.pb.go",
+    "*.svg",
+    "*CHANGELOG*",
+    "*.drawio",
+]
+[default.extend-words]
+Ded = "Ded"
```

Comment:
You can install [it](https://github.com/crate-ci/typos/) locally and simply run `typos` to see all the detected typos. This `typos.toml` will automatically be used as the configuration file for it and ignored the `extend-words` as normal words. If you remove them, they will be treated as typos and displayed in the output of `typos`. More info about this configuration file can be seen in https://github.com/crate-ci/typos/blob/master/docs/reference.md .

You can just give it a try.

Reply:
Thanks for the answer, I understand what this file does and I tried it locally, but I don't know why it makes my system stuck. Actually, I'd like to ask why the words like ND and BA were added to the ignore list as well, is it because we have the corresponding places swept out by typos? Can you give me an example? Thank you.

Reply:
Oh, I see. These strange "words" are shown in hash contents (like certificate, go.sum records, etc).

If you're not sure, you can remove them and run `typos` to see where they are.

Reply:
As for your system stuck, could you provide more information about it? I haven't seen it before.

Reply:
> Oh, I see. These strange "words" are shown in hash contents (like certificate, go.sum records, etc).

Would it be better to add these files to the ignore list?

Reply:
> > Oh, I see. These strange "words" are shown in hash contents (like certificate, go.sum records, etc).
> 
> Would it be better to add these files to the ignore list?

Yes, but they are in some test.go files which contain certificates and something like that. So I use extend-words to exclude them and these files are still being checked.

Reply:
You can add these “words” to the ignore list and also add these files to the ignore file list. What I'm worried about is the maintenance issue of this ignored list. If there is a similar word later that blocks the code integration, will the list keep changing?

Reply:
> You can add these “words” to the ignore list and also add these files to the ignore file list. What I'm worried about is the maintenance issue of this ignored list. If there is a similar word later that blocks the code integration, will the list keep changing?

If we do not consider the test file and only consider the core code area, is it possible to eliminate these strange words?

Reply:
> You can add these “words” to the ignore list and also add these files to the ignore file list. What I'm worried about is the maintenance issue of this ignored list. If there is a similar word later that blocks the code integration, will the list keep changing?

If you add the file to the ignore list, you don't need to add the words anymore. I think it's necessary to check spelling in test.go, so I added the words instead of files.

And any time you think it blocks code integration, you can add the words or files to the ignore list.

Reply:
> > You can add these “words” to the ignore list and also add these files to the ignore file list. What I'm worried about is the maintenance issue of this ignored list. If there is a similar word later that blocks the code integration, will the list keep changing?
> 
> 
> 
> If we do not consider the test file and only consider the core code area, is it possible to eliminate these strange words?

Add these affected test.go files to the ignore list and remove these words from the list.

Reply:
> If you add the file to the ignore list, you don't need to add the words anymore. I think it's necessary to check spelling in test.go, so I added the words instead of files.
> 
Files like go.sum and go.mod is unnecessary to be checked, so they should be put into the ignore list. The words like "BA" is still in the ignore list to prevent files like test.go from being checked spelling error.
> And any time you think it blocks code integration, you can add the words or files to the ignore list.

If choose this way, this file plus some comments may facilitate the understanding and modification of subsequent development.

Reply:
> Files like go.sum and go.mod is unnecessary to be checked, so they should be put into the ignore list.

Yes, but in this repository, there's no strange words found in them, so there's need to add them now.

> If choose this way, this file plus some comments may facilitate the understanding and modification of subsequent development.

Do you mean that I should add some comments for the strange words?